src/ui/public/inspector/adapters/request/request_responder.ts

```diff
@@ -43,6 +43,13 @@ export class RequestResponder {
       ...(this.request.stats || {}),
       ...stats,
     };
+    if (this.request.startTime) {
+      const startDate = new Date(this.request.startTime);
+      this.request.stats['Request timestamp'] = {
+        value: startDate.toISOString(),
+        description: 'Request start timestamp',
```

Comment:
I would also suggest making this description slightly more detailed. Something like:

`Time when the start of the request has been logged.`

Since we put this in the generic adapter I find it a bit hard what to put in here. Since we don't actually know if that's the start of the actual request e.g. in courier it's not, because it might wait a bit in the queue after the request has been logged before the request actually started. So we don't know much more than that's the time when the request has started been logging.